dataactvalidator/config/supervisord.conf

```diff
@@ -1,17 +1,18 @@
-[program:uwsgi]
-command=uwsgi --ini /data-act/backend/dataactvalidator/config/uwsgi.ini
+[program:data_act_validator_app]
+process_name=%(program_name)s_%(process_num)02d
 autostart=true
 autorestart=true
 stopasgroup=true
 killasgroup=true
+numprocs=13
 directory=/data-act/backend/dataactvalidator/
 startsecs=1
 startretries=10
-user=ec2-user
+user=root
```

Comment:
I had issues trying to start it while it was set to `ec2-user` but I can try to set it to that again